docs/configuration/config-file-reference.md

```diff
@@ -2366,6 +2366,11 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -store.cardinality-limit
 [cardinality_limit: <int> | default = 100000]
 
+# Most recent allowed cacheable result per-tenant, to prevent caching very
+# recent results that might still be in flux.
+# CLI flag: -frontend.per-user-max-cache-freshness
+[max_cache_freshness: <duration> | default = 0s]
```

Comment:
struggling with this name.  i would not have guessed the functionality based on the name.

cache_lockout_duration?  dunno.

it also kind of looks like it was already named from the diff so it may not be changeable.

Reply:
I can change the name where I'm defining the config parameter [here](https://github.com/cortexproject/cortex/pull/2609/files#diff-16c9f0ec82b1590ddc2eb9f5b5df9f51R63), but the same flag exists in results cache config as well, so the naming seems to be approved there!

Reply:
I agree this naming should match the global one we have, so I agree keeping it as is.